gene/schemas.py

```diff
@@ -112,6 +113,27 @@ class ChromosomeLocation(Location):
     chr: str
     interval: CytobandInterval
 
+    @validator('species')
```

Comment:
I think it has to match the field name
```suggestion
    @validator('species_id')
```